python-lib/text_extraction/__init__.py

```diff
@@ -134,7 +134,7 @@ def extract_markdown_chunks(markdown, filename):
         # Check each line against each of the header types (e.g., #, ##), header_level is the number of '#'
         for header_level in range(1, 4):
             # Check if line starts with a header (tab/4-spaces means code block) that we intend to split on
```

Comment:
Finally let's move the comment about tab/4 spaces next to the code related to this, lines 120/121, explaining why we don't strip these